kms/attestations/verify_attestation.py

```diff
@@ -69,3 +70,17 @@ def verify(attestation_file, bundle_file):
                 continue
         return False
 # [END verify_attestations]
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+            description=__doc__)
+    parser.add_argument('attestation_file', help="Name of attestation file.")
+    parser.add_argument('bundle_file', help="Name of certificate bundle file.")
+
+    args = parser.parse_args()
+
+    if verify(args.attestation_file, args.bundle_file):
+        print('Signature verified.')
+    else:
+        print('Signature verification failed.')
```

Comment:
Can you add a newline at the end?

Reply:
Has the linter changed? I think it warned about blank lines at the end of files last time (W391 blank line at end of file)

Reply:
@bkuang I don't think we changed it recently.